pkg/eventprocessor/eventprocessor.go

```diff
@@ -12,13 +12,15 @@ import (
 type Config struct {
 	BlockFailedExecutionBackoff time.Duration
 	DedupExecutedTxns           bool
+	HashCalcStep                int64
 }
 
 // DefaultConfig returns the default configuration.
 func DefaultConfig() *Config {
 	return &Config{
 		BlockFailedExecutionBackoff: time.Second * 10,
 		DedupExecutedTxns:           false,
+		HashCalcStep:                100,
```

Comment:
I did not add any entries to `config.json`. Pretty much relying on the default. What would be a good default? 100 feels too low. I put it 100 because it's good to see it on `staging` without waiting too much.

Reply:
Do you think we should add entries `config.json`? It seems we'd like to avoid validators messing with that config.

Reply:
The thing is that 100 blocks for Optimsim is almost "nothing" since it produces blocks at a high rate ([see here](https://optimistic.etherscan.io/txs)). 

So that would mean it will probably be doing hashing in all events. But for example in Ethereum we'd get one block per 12sec.

I'm not sure we can avoid having a knob in the config.